### DISCUSSION TITLE
Magic Lamp Effect is too fast, option should be given to change animation speed.

### DISCUSSION THREAD

**Discussion by AcidRain9** [linuxdeepin/developer-center#3007] [Features Request & Ideas | 特性请求 & 头脑风暴]
The magic lamp effect is too fast, it should be slowed down a little bit, or an option should be present for the user to choose animation speed.

**SideNote:**
Kwin has another similar effect "Yet another Magic Lamp"
Maybe the default Magic Lamp can be replaced by "Yet another Magic Lamp with Shape set to Bezier" I think it's better than "Magic Lamp"

**Reference:**
https://www.youtube.com/watch?v=4GeajayBIjc

**Comment by BLumia** [comment]
Finetuning animation speed can be done via KDE's `systemsettings` (since it's a KWin animation).

**Comment by meiyixiang** [comment]
Such functionality may not be developed for the time being; you can modify it yourself
